.github/workflows/dapr_cli.yaml

```diff
@@ -169,20 +169,15 @@ jobs:
       - name: publish binaries to github
         if: startswith(github.ref, 'refs/tags/v')
         run: |
-          echo "installing github-release-cli..."
-          sudo npm install --silent --no-progress -g github-release-cli@1.3.1
           # Get the list of files
           RELEASE_ARTIFACT=(${ARTIFACT_DIR}/*)
-          # Parse repository to get owner and repo names
-          OWNER_NAME="${GITHUB_REPOSITORY%%/*}"
-          REPO_NAME="${GITHUB_REPOSITORY#*/}"
           export GITHUB_TOKEN=${{ secrets.DAPR_BOT_TOKEN }}
           echo "Uploading Dapr CLI Binaries to GitHub Release"
-          github-release upload \
-            --owner $OWNER_NAME --repo $REPO_NAME \
-            --tag "v${REL_VERSION}" \
-            --name "Dapr CLI v${REL_VERSION}" \
-            --prerelease true \
+          gh release create \
```

Comment:
Is this verfied ? See few of the parameters are not present here, may be not required with gh cli ?

Reply:
Looks like based on the documentation. 
1. `--owner` and `--repo` are combined into `--repo`
2. `--tag` is present has an argument to the create command.
3. `--name` has become `--title`